_includes/templates/install/cassandra-ubuntu-install.md

```diff
@@ -1,52 +1,23 @@
-Instructions listed below will help you to install Cassandra.
+The following commands will install Apache Cassandra and its command-line tools:
 
 ```bash
-# Add cassandra repository
-echo "deb https://debian.cassandra.apache.org 41x main" | sudo tee /etc/apt/sources.list.d/cassandra.sources.list
-curl https://downloads.apache.org/cassandra/KEYS | sudo apt-key add -
+# Add Cassandra repository
+echo "deb [signed-by=/etc/apt/keyrings/apache-cassandra.asc] https://debian.cassandra.apache.org 50x main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+sudo curl -o /etc/apt/keyrings/apache-cassandra.asc https://downloads.apache.org/cassandra/KEYS
 sudo apt-get update
+
 ## Cassandra installation
-sudo apt-get install cassandra
-## Tools installation
-sudo apt-get install cassandra-tools
+sudo apt-get install cassandra cassandra-tools
 ```
+{: .copy-code}
 
-In order to run Cassandra, you will need to install Java 11.
+To verify that Cassandra is running, check the node status:
 
-Install the package.
```

Comment:
Cassandra 5 depends on 17 java, so it is ok. Might not work in old Ubuntu distributions, but it is ok